lib/rules/lint-no-whitespace-for-layout.js

```diff
@@ -0,0 +1,26 @@
+// lint-no-whitespace-for-layout.js
+'use strict';
+const Rule = require('./base');
+
+const ERROR_MESSAGE = 'Excess whitespace detected.';
+
+module.exports = class noWhitespaceForLayout extends Rule {
+  visitor() {
+    return {
+      TextNode(node) {
+        let source = this.sourceForNode(node);
+        let matches = source.match(/(( )|(\&nbsp\;))(( )|(\&nbsp\;))/g);
```

Comment:
Can you put a comment here to say what this is doing? I'm thinking something like:

```
// check for two ` ` or `&nbsp;` in a row
```